pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "times-excel-reader"
+version = "0.1.0"
+description = 'An open source tool to convert TIMES model Excel input files to DD format ready for processing by GAMS'
+readme = "README.md"
+requires-python = ">=3.8"
+license = { file = "LICENSE" }
+keywords = []
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "License :: OSI Approved :: MIT License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+]
+dependencies = [
+  "GitPython >= 3.1.31, < 3.2",
+  "more-itertools",
+  "openpyxl >= 3.0, < 3.1",
+  "pandas >= 1.5",
+]
+
+[project.optional-dependencies]
+dev = ["black", "pre-commit", "tabulate"]
+
+[project.urls]
+Documentation = "https://github.com/etsap-TIMES/times-excel-reader#readme"
+Issues = "https://github.com/etsap-TIMES/times-excel-reader/issues"
+Source = "https://github.com/etsap-TIMES/times-excel-reader"
+
+[project.scripts]
+times-excel-reader = "times_reader.__main__:main"
```

Comment:
@olejandro this is why what we were trying today didn't work -- the entry point for the `times-excel-reader` script/command is the `main` function, so modifying the `if __name__ == "__main__"` block had no effect!